From f847f1cf13c9d15bc221bb1dca43ba885b7e33b0 Mon Sep 17 00:00:00 2001
From: Li Zhou <li.zhou@windriver.com>
Date: Fri, 26 Nov 2021 18:52:15 +0800
Subject: [PATCH 2/4] kernel-rt: Add a new changelog file

Don't list the commits logs and only give out the link for checking
about the git history of this source code.

Signed-off-by: Li Zhou <li.zhou@windriver.com>
---
 debian/changelog | 7 +++++++
 1 file changed, 7 insertions(+)
 create mode 100644 debian/changelog

diff --git a/debian/changelog b/debian/changelog
new file mode 100644
index 0000000..e78bf2a
--- /dev/null
+++ b/debian/changelog
@@ -0,0 +1,7 @@
+linux-rt (5.10.74-1) unstable; urgency=medium
+
+  * New upstream update:
+    https://git.yoctoproject.org/linux-yocto/log/?h=v5.10%2Fstandard%2Fpreempt-rt%2Fbase&qt=range&q=8048444fc7e44c6fc04a5e0b35d6117fe51c17fc
+
+ -- Li Zhou <li.zhou@windriver.com>  Tue Nov 29 10:22:35 CST 2021
+
-- 
2.17.1

